kbase-extension/static/kbase/js/util/timeFormat.js

```diff
@@ -298,6 +299,22 @@ define([], function() {
         return pluralizeTimeStr(Math.floor(seconds), 'second');
     }
 
+    // There was a request that the staging panel tighten up the display a little bit, which was using getTimeStampStr to format the age.
+    // So this function will just take the output of regular getTimeStampStr and shorten it up a little bit.
+    // * Replace "long" durations with shortened ones.
+    // * drop " ago" from the end.
+
+    function getShortTimeStampStr (objInfoTimeStamp, alwaysExact) {
```

Comment:
A few tests should be added here. Check out `test/unit/spec/Util/timeFormatSpec.js`

Reply:
I added in several tests to verify the new function.

While I was at it, I discovered that the existing getTimeStampStr would favor a larger interval of smaller units if it was divisible by one.
i.e., instead of saying "1 day ago" it would be "24 hours ago". Or 1 hour -> 60 minutes, 1 minute -> 60 seconds.

I re-tooled that to favor smaller increments of the larger interval, and added a few more tests to confirm that behavior as well, on both functions.